packages/react-static/src/utils/binHelper.js

```diff
@@ -114,8 +114,16 @@ ignoredExtensions.forEach(ext => {
   require.extensions[`.${ext}`] = () => {}
 })
 
-console.error = (err, ...rest) => {
-  console.log(new PrettyError().render(err), ...rest)
+console.error = (...args) => {
+  if(args.length === 0){
+    return console.log()
+  }
+  const [err, ...rest] = args
+  if(err){
+    console.log(new PrettyError().render(err), ...rest)
+  } else {
+    console.log(err, ...rest)
+  }
```

Comment:
I checked [`pretty-error`](https://github.com/AriaMinaei/pretty-error) and it really only handled errors.

```suggestion
  if(err instanceof Error){
    console.log(new PrettyError().render(err), ...rest)
    return
  }
  
  return originalConsoleError(err, ...rest)
```